frontend/src/app/projects/create-project/create-project.component.html

```diff
@@ -3,14 +3,14 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-<div class="wrapper">
-  <mat-horizontal-stepper linear #stepper>
-    <mat-step completed="false" label="General information">
-      <div class="flex-center">
-        <mat-card class="section-card">
+<div class="wrapper flex justify-center">
```

Comment:
The mobile view has room for improvement. It's wasn't perfect on main either. To do so, you can add a few `max-w-mobile md:w-full` classes and add `mobile` to the `spacing` section of the `tailwind.config.js`. 
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/f91e2114-d28e-4862-b80b-463ae8ca15ac)

Reply:
This one I'm kind of at a loss for. I don't think there's any way to continue using the horizontal stepper on mobile, so maybe I can flip it into a vertical stepper at a certain breakpoint? Removing the back buttons made sense on desktop but they're really important for mobile, but since we sometimes skip steps I don't know if that would require special logic, or do we leave them off?

Reply:
Vertical also didn't look great, so I opted for horizontal with hidden labels and conditionally displayed headers so the user knew what step they were on. There's a weird media query for the stepper-lines and stepper headers, but apparenlty you can only target that with CSS, unless you know a workaround.
![mobile](https://github.com/DSD-DBS/capella-collab-manager/assets/79602065/9c1ad0b7-8ff5-47bb-abc2-4d32add3f9cc)

Reply:
I really like the new mobile design, thanks for the implementation and the idea 👍